OpenOversight/tests/test_models.py

```diff
@@ -137,28 +139,33 @@ def test_assignment_repr(mockdata):
     assignment = Assignment.query.first()
     assert (
         repr(assignment)
-        == f"<Assignment: ID {assignment.base_officer.id} : {assignment.star_no}>"
+        == f"<Assignment ID: {assignment.base_officer.id} : {assignment.star_no}>"
     )
 
 
+def test_incident_repr(mockdata):
+    incident = Incident.query.first()
+    assert repr(incident) == f"<Incident ID: {incident.id} : {incident.report_number}>"
+
+
 def test_job_repr(mockdata):
     job = Job.query.first()
     assert repr(job) == f"<Job ID {job.id}: {job.job_title}>"
```

Comment:
Should the format for Job ID also change? Since this looks more like the other format.